projects/aca-content/src/lib/components/search/search-results/search-results.component.scss

```diff
@@ -1,6 +1,7 @@
 @import '../../../ui/mixins';
 
 $adf-chip-background: #efefef;
+$adf-facets-background: white;
```

Comment:
If you open browser's console and scroll down you can see there is list of already defined variables: 
<img width="398" alt="image" src="https://github.com/Alfresco/alfresco-content-app/assets/115619721/33686755-0834-44e7-b559-bc8e4106d0a0">

Some of them have already white value. Could you please verify if we can reuse them or you need create new one?

Reply:
Yep, I saw those variables, but I was not sure if I should use them, what do you think?

Reply:
I would say it is good to reuse existing assuming that its name matches to your case. For example if there is any variable named "modal" then this does not match to your case but if there are some more generic name then should be fine in my opinion. 

Generally variables are helpful to share style values between places so then we can change style for each place by changing just single value. If we define separate variable for each place then we lose that advantage because we need then change each place instead of changing just one place